.github/workflows/integration-tests.yml

```diff
@@ -111,42 +117,43 @@ jobs:
 
       ## Latest Version Tests
       ## Runs the plugin tests against the latest stable version of Grafana.
-      - name: Build plugin with latest version of e2e libraries
-        run: |
-          yarn upgrade @grafana/e2e-selectors @grafana/e2e --latest
-        working-directory: ${{ matrix.pluginDir }}
+      ## Activate this tests again once grafana/grafana sorts out the issue with npm packages mismatching the docker image tags
```

Comment:
Can we uncomment this and try running it again please? Turns out the npm latest dist-tag was pointing at the 9.3.0-beta1. 🤦

Reply:
Uncommented https://github.com/grafana/grafana-plugin-examples/pull/112/commits/5022ba636ed645041e6438a3540f145d1adab445